### ISSUE TITLE
not installable on ubuntu 23.04 lunar

### ISSUE THREAD

**Issue by banteg** [MisterTea/EternalTerminal#579]
```
$ sudo apt install et
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 et : Depends: libprotobuf-lite23 (>= 3.12.4) but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by duraki** [comment]
Same for Debian 12 which corresponds to 22.04. @MisterTea can you point us in right direction when it comes to these deps that aren't in foss sources?

**Comment by MisterTea** [comment]
Hi!  The problem is that the protobuf team eliminated protobuf-lite.  I'll
need to switch et to use protobuf instead of the lite version. Will do this
next week.

**Comment by duraki** [comment]
Awesome. Thanks a lot for all of your contribution. I just started using it on my `staging` and I love it. Do you think its production ready?

**Comment by prldm** [comment]
I got this when trying to build it from source:
`E: Package 'libcurl4-dev' has no installation candidate`
since they removed this package in 23.04, I had to install libcurl4-openssl-dev instead. please fix this.

**Comment by MisterTea** [comment]
Fixed in 6.2.8
